src/backend/expungeservice/record_editor.py

```diff
@@ -102,7 +103,7 @@ def _add_charge(case_number, ambiguous_charge_id, edit) -> Charge:
             "case_number": case_number,
             "id": f"{ambiguous_charge_id}-0",
             "statute": "",
-            "level": "",
```

Comment:
What happens if you just remove `"level": "",` instead of doing the change you made here? Would "level" then not be defined sometimes? In which case what would it default to?

Reply:
Ah yes, it's required in the frontend like everything else so I've removed the default here and added a value to all the unit tests.